README.md

```diff
@@ -8,6 +8,34 @@ and that meet our community's quality bar.
 One of the cool things about trusted libraries is that we take care of building, testing and
 publishing them for you.
 
+<!-- WINGLIBS_TOC_START -->
+
+## Table of contents
```

Comment:
Clever header suggestions welcome ("the humble winglibs catalog" etc.)

Reply:
"All the libs"?

Reply:
For political reasons I'll pass on this one 😆